src/Apps/Legion/Services/WizardTypes.utils.ts

```diff
@@ -11,19 +11,35 @@ import {
     IViewType
 } from '../Models';
 
+// #region Base
+
+function initializeBase(args: {
```

Comment:
in fact this is not initialization I think, just fetching base properties of objects. I mean seeing ...initializeBase(something) makes me think if it is setting initial values.
```suggestion
function getBase(args: {
```

Reply:
Agreed. Done.